### PR TITLE
add facetime extension

### LINKED ISSUE #869: New extension to Make calls from RayCast
Making calls from Macos is pretty tough, you have to open Facetime and there is not a easy interface to enter the phone number and call. Would be very awesome to make a simple extension where you just put the number and trigger the "make call" command.

### PR DESCRIPTION
**Pull request by ron-myers**
## Description

Add Facetime extension that can setup calls in Facetime.  Closes #869 

## Screencast

https://user-images.githubusercontent.com/10678906/153772266-9745e451-dc8b-4412-8f85-19384a33dd6d.mov

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)

### CODE REVIEW

**Comment by ron-myers** [comment]
I have the start of a contact search to add to this but will wait until after this is reviewed / accepted to layer in that feature.

File: extensions/facetime/package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  "title": "Facetime",
+  "description": "Make Facetime calls with Raycast",
+  "icon": "facetime.png",
+  "author": "ron-myers",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "call",
+      "title": "Facetime Call",
+      "subtitle": "Facetime",
```
Comment (pernielsentikaer) [inline extensions/facetime/package.json:13]:
```suggestion
```
Redundant subtitle

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @ron-myers 👋 

I have now checked your contribution and I have some feedback for you:

- The provided logo in the extension is not provided in 512×512 pixels. Please modify this according to [Extension Icon guideline](https://developers.raycast.com/basics/prepare-an-extension-for-store#extension-icon)

- Instead of a form with a default value, it would be cool if was a list instead where it was possible to add/remove “contacts” 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

File: extensions/facetime/package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "facetime",
+  "title": "Facetime",
```
Comment (pernielsentikaer) [inline extensions/facetime/package.json:4]:
```suggestion
  "title": "FaceTime",
```

File: extensions/facetime/package.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  "description": "Make Facetime calls with Raycast",
```
Comment (pernielsentikaer) [inline extensions/facetime/package.json:5]:
```suggestion
  "description": "Make FaceTime calls with Raycast",
```

File: extensions/facetime/package.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  "icon": "facetime.png",
+  "author": "ron-myers",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "call",
+      "title": "Facetime Call",
```
Comment (pernielsentikaer) [inline extensions/facetime/package.json:12]:
```suggestion
      "title": "FaceTime Call",
```

File: extensions/facetime/package.json [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+      "description": "Opens Facetime ready to make a call",
```
Comment (pernielsentikaer) [inline extensions/facetime/package.json:13]:
```suggestion
      "description": "Opens FaceTime ready to make a call",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @ron-myers 👋 

Sounds like a plan, and yes, we can add more functionality in a later version for sure 💪 

What I would like to get added in this version is a preference to set a default number instead, then it's possible to leave blank too 😄 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

File: extensions/facetime/src/call.tsx [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+  return (
+    <Form
+      actions={
+        <ActionPanel>
+          <Action.SubmitForm onSubmit={handleSubmit} title="Setup Call" />
+        </ActionPanel>
+      }
+    >
+      <Form.TextField id="number" title="Number" placeholder="Enter text" defaultValue="" />
```
Comment (pernielsentikaer) [inline extensions/facetime/src/call.tsx:33]:
```suggestion
      <Form.TextField id="number" title="Number" placeholder="Enter number" defaultValue="" />
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @ron-myers 👋 

Last one to change before we can accept the first version 🔥 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by ron-myers** [comment]
Appreciate all of your feedback @pernielsentikaer

**Review by pernielsentikaer — APPROVED** [review]
Hi @ron-myers 🔥 

Thanks for the corrections, it's a very nice first version—already looking forward to the next version 😄 

Accepted! 🚀

**Review by mathieudutour — APPROVED** [review]
Looking good :+1:

**Comment by afanjul** [comment]
Hi @ron-myers , I have tested the app you did (catching my suggestion) but my problem is that the Call is not made using my iPhone, but using FaceTime... My first intention (and suggestion) was to have an easy way to call somebody with a "normal voice call" using my iPhone (and Handoff technology). I suggested that, because in deed it's pretty awful (poor UX) to make it through FaceTime.
Time ago I had a "dial pad" app that worked pretty well: https://www.theverge.com/2014/10/21/7028427/the-phone-dialer-apple-should-have-included-with-yosemite but now I think it's outdated, and I only can easily take calls from my Mac, but not make them...

could you add that feature?
thanks

**Comment by pernielsentikaer** [comment]
Hi @afanjul 👋 

Could you please add it as an [API Feature Request](https://github.com/raycast/extensions/issues/new?assignees=&labels=api%2C+feature+request&template=api_feature_request.md&title=), so it's not left just here 😄

**Pull request merged**